### PR TITLE
media-fonts/b612-font: New package

### PR DESCRIPTION
**Pull request by janhenke**
Signed-off-by: Jan Henke <Jan.Henke@taujhe.de>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+inherit font vcs-snapshot
+
+DESCRIPTION="Font designed for aircraft cockpit displays"
+HOMEPAGE="http://b612-font.com/"
+SRC_URI="https://git.polarsys.org/c/b612/b612.git/snapshot/b612-bd14fde2544566e620eab106eb8d6f2b7fb1347e.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```
Comment (juippis) [inline media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: I don't think you can add arm and arm64 keywords to new packages.

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+IUSE=""
+
+FONT_S="${S}/TTF"
+FONT_SUFFIX="ttf"
+
+DOCS="README.md"
```
Comment (juippis) [inline media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: This should be part of the default installation, and you should be able to remove this line. If it doesn't work, then at least use the array form instead, `DOCS=( )`.

File: media-fonts/b612-font/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@taujhe.de</email>
+		<name>Jan Henke</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>B612 is an highly legible open source font family designed and tested to be used on aircraft cockpit screens.</longdescription>
```
Comment (juippis) [inline media-fonts/b612-font/metadata.xml]: Please make this a multi-liner.

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild (hunk 1 above)
Comment (janhenke) [reply media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: Too bad

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild (hunk 2 above)
Comment (janhenke) [reply media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: When leaving out this line, the file is not installed. Changed it into an array.

File: media-fonts/b612-font/metadata.xml (hunk 3 above)
Comment (janhenke) [reply media-fonts/b612-font/metadata.xml]: Done

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+FONT_S="${S}/TTF"
+FONT_SUFFIX="ttf"
+
+DOCS=(
+	"README.md"
```
Comment (a17r) [inline media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: no need for quotes and multiline

File: media-fonts/b612-font/b612-font-1.003_p20171129.ebuild (hunk 1 above)
Comment (a17r) [reply media-fonts/b612-font/b612-font-1.003_p20171129.ebuild]: you can, if you can test on arm/arm64 yourself.

File: media-fonts/b612-font/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	<maintainer type="person">
+		<email>gentoo@taujhe.de</email>
+		<name>Jan Henke</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		B612 is an highly legible open source font family designed
```
Comment (a17r) [inline media-fonts/b612-font/metadata.xml]: `s/an/a/`

**Pull request closed without merging**